vendor/github.com/gophercloud/gophercloud/starlingx/inventory/v1/storagebackends/requests.go

```diff
@@ -16,6 +16,7 @@ type StorageBackendOpts struct {
 	State        *string                 `json:"state,omitempty" mapstructure:"state"`
 	Task         *string                 `json:"task,omitempty" mapstructure:"task"`
 	Services     *string                 `json:"services,omitempty" mapstructure:"services"`
+	Network      *string                 `json:"network,omitempty" mapstructure:"network"`
```

Comment:
It looks like you've manually modified these gophercloud files here, rather than following the section of the developer doc that deals with pulling in changes to this pkg:
https://github.com/Wind-River/cloud-platform-deployment-manager/blob/master/DEVELOPER.md#working-with-a-private-fork-of-a-vendor-package

Reply:
Seems 'dep ensure' updates Gopkg.lock with the commit hash of the local repo. Will wait until gophercloud commit merges, so it picks up the correct upstream commit hash instead of the local branch one.